app/controllers/api/v1/tickets_controller.rb

```
@@ -512,32 +494,32 @@ def debug_ticket_counts
         end
-      
+
         scope = @organization.tickets
-      
+
         scope = scope.where(status: params[:status]) if params[:status].present?
         scope = scope.where(ticket_type: params[:ticket_type]) if params[:ticket_type].present?
-      
+
         if params[:start_date].present? && params[:end_date].present?
           start_date = Date.parse(params[:start_date]) rescue nil
```

RuboCop (1.76.1)

[convention] Avoid using `rescue` in its modifier form.

(Style/RescueModifier)